addons/web/static/src/legacy/scss/daterangepicker.scss

```diff
@@ -74,14 +124,29 @@
     .calendar-time select {
         &.hourselect, &.minuteselect, &.secondselect, &.ampmselect {
             display: initial;
+            border-color: var(--daterangepicker__select-border-color);
+            background-color: var(--daterangepicker__select-bg);
+            color: var(--daterangepicker__select-color);
             -webkit-appearance: menulist-button;
             -moz-appearance: menulist-button;
             appearance: menulist-button;
         }
     }
 
-    .drp-buttons .drp-selected {
-        display: none;
+    .drp-buttons {
+        border-top: 0;
+
+        .btn {
+            text-transform: uppercase;
+
+            // Hard-coded 'cause `$btn-font-weight` returns inconsistent values
```

Comment:
could you add a bit more explanation for this issue?

Reply:
Thanks to system-fonts it can be now solved without a workaround https://github.com/odoo/odoo/pull/99755/files#diff-708b8e75f1eab565b90134306951450e5d56eb34961d8889fb86a9a0b1d371c9R141